src/main/java/teammates/common/datatransfer/FeedbackRankOptionsQuestionDetails.java

```diff
@@ -273,15 +262,15 @@ public String getQuestionResultStatisticsHtml(
 
             String option = entry.getKey();
             
-            fragments.append(Templates.populateTemplate(FeedbackQuestionFormTemplates.RANK_RESULT_STATS_OPTIONFRAGMENT,
-                                                                        "${rankOptionValue}", Sanitizer.sanitizeForHtml(option),
+            fragments.append(Templates.populateTemplate(FormTemplates.RANK_RESULT_STATS_OPTIONFRAGMENT,
+                    Slots.RANK_OPTION_VALUE,  Sanitizer.sanitizeForHtml(option),
                                                                         "${ranksReceived}", ranksReceived,
                                                                         "${averageRank}", df.format(average)));
```

Comment:
I'm guessing previous level of indentation will bust the line length limit. In that case, align the other two parameters below.